components/event_dispatcher/introduction.rst

```diff
@@ -343,30 +299,31 @@ method notifies all listeners of the given event. It takes two arguments:
 the name of the event to dispatch and the ``Event`` instance to pass to
 each listener of that event::
 
-    use Acme\StoreBundle\StoreEvents;
-    use Acme\StoreBundle\Order;
-    use Acme\StoreBundle\Event\FilterOrderEvent;
+    use Acme\Store\Order;
+    use Acme\Store\Event\OrderPlacedEvent;
 
     // the order is somehow created or retrieved
     $order = new Order();
     // ...
 
-    // create the FilterOrderEvent and dispatch it
-    $event = new FilterOrderEvent($order);
-    $dispatcher->dispatch(StoreEvents::STORE_ORDER, $event);
+    // create the OrderPlacedEvent and dispatch it
+    $event = new OrderPlacedEvent($order);
+    $dispatcher->dispatch(OrderPlacedEvent::NAME, $event);
 
-Notice that the special ``FilterOrderEvent`` object is created and passed
-to the ``dispatch`` method. Now, any listener to the ``store.order`` event
-will receive the ``FilterOrderEvent`` and have access to the ``Order`` object
-via the ``getOrder`` method::
+Notice that the special ``OrderPlacedEvent`` object is created and passed to
+the ``dispatch`` method. Now, any listener to the ``store.order_placed`` event
+will receive the ``OrderPlacedEvent``::
```

Comment:
It seems we are missing to describe how to register the listener (we only document the subscriber below).